test/contributionreward.js

```diff
@@ -84,15 +84,15 @@ contract('ContributionReward', function(accounts) {
       assert.equal(tx.logs[0].event, "LogNewContributionProposal");
      });
 
-     it("submitContribution fees", async function() {
+     it("proposeContributionReward fees", async function() {
        var testSetup = await setup(accounts,14,10);
 
        // await testSetup.contributionReward.registerOrganization(testSetup.org.avatar.address);
```

Comment:
Though this PR did not update that ..
I suggest  to remove this commented code.